### ISSUE TITLE
Export webpack files in package.json

### ISSUE THREAD

**Issue by justrhysism** [single-spa/single-spa-angular#408]
Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂

Today I used [patch-package](https://github.com/ds300/patch-package) to patch `single-spa-angular@6.0.0` for the project I'm working on.

The angular template generates a file `extra-webpack.config.js` which deep-requires the webpack config:

```ts
const singleSpaAngularWebpack = require('single-spa-angular/lib/webpack').default;
``` 

However, this isn't exposed by the `single-spa-angular` `package.json` which results in this error:

```log
fatal: Unable to read current working directory: No such file or directory
[error] Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './lib/webpack' is not defined by "exports" in /path/to/app/node_modules/single-spa-angular/package.json
    at new NodeError (node:internal/errors:371:5)
    at throwExportsNotFound (node:internal/modules/esm/resolve:440:9)
    at packageExportsResolve (node:internal/modules/esm/resolve:692:3)
    at resolveExports (node:internal/modules/cjs/loader:482:36)
    at Function.Module._findPath (node:internal/modules/cjs/loader:522:31)
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:919:27)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/path/to/app/extra-webpack.config.js:1:33)
```

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/single-spa-angular/package.json b/node_modules/single-spa-angular/package.json
index 73ae5b0..5c852e7 100644
--- a/node_modules/single-spa-angular/package.json
+++ b/node_modules/single-spa-angular/package.json
@@ -70,6 +70,10 @@
       "es2015": "./fesm2015/single-spa-angular-parcel.mjs",
       "node": "./fesm2015/single-spa-angular-parcel.mjs",
       "default": "./fesm2020/single-spa-angular-parcel.mjs"
+    },
+    "./lib/webpack": {
+      "types": "./lib/webpack/index.d.ts",
+      "default": "./lib/webpack/index.js"
     }
   },
   "dependencies": {
```

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by justrhysism** [comment]
Okay, just took at the PRs and see that this is addressed by #405

**Comment by arturovt** [comment]
https://github.com/single-spa/single-spa-angular/releases/tag/v6.0.2.
